src/client/Microsoft.Identity.Client/Platforms/Android/Broker/BrokerRequest.cs

```diff
@@ -65,7 +65,17 @@ public Uri RedirectUri
             AcquireTokenInteractiveParameters acquireTokenInteractiveParameters)
         {
             BrokerRequest br = FromAuthenticationParameters(authenticationRequestParameters);
-            br.Prompt = acquireTokenInteractiveParameters.Prompt.PromptValue.ToUpperInvariant();
+            var prompt = acquireTokenInteractiveParameters.Prompt;
+
+            if (prompt == Client.Prompt.NoPrompt || prompt == Client.Prompt.NotSpecified)
+            {
+                br.Prompt = "NONE";
```

Comment:
Isn't this a breaking change?

Reply:
we used to default to "none" for broker a long time ago but I think the behavior for "none" and "select_account" is ultimately the same. Ill switch to select account.